src/main/resources/templates/fragments/inlineStyles.html

```diff
@@ -0,0 +1,17 @@
+<div th:fragment="styles">
+    <style>
+        body {
+            min-height: 75rem;
+            padding-top: 4.5rem;
+            padding-left: 10px;
+        }
+        label {
+            width: 180px;
+            clear: left;
+            text-align: left;
+        }
+        input, label {
+            float: left;
+        }
+    </style>
+</div>
```

Comment:
Created a new fragment for the style tags that were originally in the index. This could be equivalently produced using another stylesheet, but for the sake of how the project was set up, I wanted to keep this parallel. In the future this will provide significantly less technical debt, as well as help you avoid duplication listed in Baeldung's Java Clean Code article. 

Source: https://martinfowler.com/bliki/TechnicalDebt.html
Source: https://www.baeldung.com/java-clean-code